src/configs/git_status.rs

```diff
@@ -0,0 +1,46 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
```

Comment:
Also doing this in #434

Reply:
Okay, cool. I've reverted my changes to the `git_status` module.